postgresql/manifests/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- postgres.yaml
+- backup.yaml
+- backup-pvc.yaml
+configMapGenerator:
+- files:
+  - pg_hba.conf=pg_hba.conf
```

Comment:
I think it would be a good idea for the the `pg_hba.conf` and `pg_init_users.sh` to be included just like the scripts are in the Cockroachdb-manifests repo. 
Whether as examples of as part of the base, I think it'll be easier for newcomers to get to grips with if they have something to look at

Reply:
Ahh poop I removed these files all together. I dont think we use them for anything

Reply:
In that case it's easier then 😛